api_tests/lib/ethereum.ts

```diff
@@ -81,31 +84,36 @@ export class EthereumWallet {
         return this._address;
     }
 
-    async fund(ethAmount: BN) {
+    async fund(ethAmount: BN | string) {
         const parity_dev_account = "0x00a329c0648769a73afac7f9381e08fb43dbea72";
         const parity_dev_password = "";
-        const weiAmount = _web3Client.utils.toWei(ethAmount, "ether");
+
+        const weiAmount = utils.toWei(ethAmount, "ether");
         const tx = {
             from: parity_dev_account,
             to: this.address(),
-            value: _web3Client.utils.numberToHex(weiAmount),
+            value: utils.numberToHex(weiAmount),
         };
         return _web3Client.eth.personal.sendTransaction(
             tx,
             parity_dev_password
         );
     }
 
-    async send_eth_transaction_to(
+    async sendEthTransactionTo(
         to: string,
         data: string = "0x0",
-        value: string = "0",
+        value: BN | string | number = utils.toBN(0),
```

Comment:
Does this mean this function accepts either `BN` or `string` or defaults to `number = ...` ?

Reply:
accepts `BN` or `string` or `number` and default to `utils.toBN(0)` which is a `BN`